base/context.go

```diff
@@ -58,13 +64,14 @@ type Context struct {
 
 // InitDefaults sets up the default values for a context.
 func (ctx *Context) InitDefaults() {
+	ctx.Insecure = defaultInsecure
```

Comment:
I know, I'm just following the pattern of Context fields being initialized by defaultX